### DISCUSSION TITLE
deploy from github

### DISCUSSION THREAD

**Discussion by ghost** [deta/cloud-docs#478] [Ideas]
I want to deploy from github. When I develop outside of my PC, it would be useful to be able to deploy from github.

**Comment by jnsougata** [comment]
That's possible. Push all your codes to GitHub repo and most importantly `.deta` folder too and set a github workflow.
You will find a community built workflow  at: https://github.com/marketplace/actions/deploy-to-deta 
It's safe to use.

**Reply by lemonyte** [reply]
Including the `.deta` folder is unnecessary because the workflow simply clones the micro, and the state file will add bloat to your commit history.

**Reply by jnsougata** [reply]
Interesting…. just notice
